packages/webui/src/client/lib/rundownTiming.ts

```diff
@@ -113,24 +113,24 @@ export class RundownTimingCalculator {
 		let displayStartsAtAccumulator = 0
 		let segmentDisplayDuration = 0
 		let segmentBudgetDurationLeft = 0
+		let remainingBudgetOnCurrentSegment: undefined | number
 
 		const rundownExpectedDurations: Record<string, number> = {}
 		const rundownAsPlayedDurations: Record<string, number> = {}
 
 		let rundownsBeforeNextBreak: Rundown[] | undefined
 		let breakIsLastRundown: boolean | undefined
-		let liveSegmentId: SegmentId | undefined
+		let liveSegmentIds: [SegmentId, SegmentPlayoutId] | undefined
```

Comment:
I'm unsure about this tuple setup - it seems unusual, and I feel like an ad-hoc object would be more natural here? `liveSegmentIds[1]` is quite opaque, and I would expect this to be "a list of LiveSegmentIds" and not "Live  Segment's Ids".